test/javascript/playingTheGame_test.js

```diff
@@ -256,10 +256,10 @@ describe('Playing the game', () => {
   const pickupCard = (cardPos, ownershipPromise, objectLocator) => {
     if(cardPos < 0) cardPos = initialGameState.cards.length + cardPos;
     const card = initialGameState.cards[cardPos];
-    if(!objectLocator) {
-      let cards = getCards(card.stackId);
-      objectLocator = cards[cards.length - 1].objectLocator;
-    }
+    // if(!objectLocator) {
```

Comment:
need to work out a different way to get the object Locator or of doing this test as getCards no longer exists